gcpy/cstools.py

```diff
@@ -692,10 +689,113 @@ def is_cubed_sphere(
     is_gchp : bool
         Returns True if data is placed on a cubed-sphere grid,
         and False otherwise.
+
+    Remarks:
+    --------
+    A cubed-sphere data file has one of the following attributes
+    (1) A dimension named "nf" (GCHP/GEOS diagnostic files)
+    (2) The lat/lon ratio is exactly 6 (GCHP/GEOS checkpoints)
+    """
+    gcpy.util.verify_variable_type(data, (xr.DataArray, xr.Dataset))
+
+    if is_cubed_sphere_diag_grid(data):
+        return True
+    if is_cubed_sphere_rst_grid(data):
+        return True
+    return False
+
+
+def is_cubed_sphere_diag_grid(data):
+    """
+    Determines if a cubed-sphere grid has History file dimensions.
+    (i.e. a dimension named "nf", ak number of grid faces).
+
+    Args:
+    -----
+    data : xarray.DataArray or xarray.Dataset
+        The input data.
+
+    Returns:
+    --------
+    True if the grid has restart dimensions, False otherwise.
```

Comment:
Is this supposed to be if the grid has "nf" dimensions rather than restart dimensions?

Reply:
@lizziel: Now fixed in commit 45009436.